provider/pkg/await/ingress.go

```diff
@@ -336,21 +346,27 @@ func (iia *ingressInitAwaiter) checkIfEndpointsReady() (string, bool) {
 		}
 
 		for _, rule := range obj.Spec.Rules {
-			if rule.HTTP == nil {
-				iia.config.logStatus(diag.Error, fmt.Sprintf("expected value %q is unset for ingress: %s",
-					".spec.rules[*].http", obj.Name))
-				return apiVersion, false
+			var httpRules []networkingv1beta1.HTTPIngressPath
```

Comment:
Are we unmarshalling to the `v1beta1` struct? I would have expected to use the `v1` struct.

Reply:
I guess it would depend on the preferred version in the kubernetes version? I do agree with you but lets keep this for now and reconsider at the next major release.